shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthSessionItem.java

```diff
@@ -147,4 +169,22 @@ public AuthSessionItem withEmailAddress(String emailAddress) {
         this.emailAddress = emailAddress;
         return this;
     }
+
+    public String toLogSafeString() {
```

Comment:
Would you be happy to add a comment to this method making it clear that it can't contain PII please? I initially assumed this means it only contained loggable characters...

Reply:
Done